### ISSUE TITLE
f.e21.FWscHIST.ne30_L48_BL10_cam6_3_046_control.hf.001

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#64]
**Description:**

Identical to https://github.com/NCAR/amwg_dev/issues/53 but using latest tag cam6_3_046. 
This run invokes the new pumas microphysics provided you set -phys cam_dev. 

Extra namelist setting: 
ncdata = '/glade/p/cesm/amwg_dev/juliob/FWsc_ne30pg3_58L_GRID_48_taperstart10km_lowtop_BL10_v3_beta1p75_Top_43km.nc'
micro_mg_precip_fall_corr = .true.
se_rsplit            = 3
se_nsplit           = 2
se_hypervis_subcycle =  4

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/$f.e21.FWscHIST.ne30_L48_BL10_cam6_3_046_control.hf.001

On github (if available):
https://github.com/NCAR/amwg_dev/tree/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_046_control.hf.001

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cam6_3_046

On github (if available):
https://github.com/ESCOMP/CAM/tree/cam6_3_046
hash: 332a90d

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/FWscHIST/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_046_control.hf.001/atm/](https://webext.cgd.ucar.edu/FWscHIST/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_046_control.hf.001/atm/)

**Contacts:**
@JulioTBacmeister
@adamrher
@PeterHjortLauritzen 
@andrewgettelman 
@cecilehannay

**Comment by cecilehannay** [comment]
I processed the diags on 5 years for the PUMAS run:
https://webext.cgd.ucar.edu/FWscHIST/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_046_control.hf.001/atm/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_046_control.hf.001.1980_1984-f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_control.hf.001.1980_1989/

Given the large differences, I don't think we need more than 5 years to look at this run.

**Comment by andrewgettelman** [comment]
Yep, thinner clouds. Same as in my experiments. Now I suppose you are going to want me to tune them....

**Comment by cecilehannay** [comment]
> Yep, thinner clouds. Same as in my experiments. Now I suppose you are going to want me to tune them....
You are reading my mind. ;)

But more seriously: I was just wondering if we need 10 years - given the differences. I thought 5 years could be enough. 
But I could do 10 years too.

**Comment by JulioTBacmeister** [comment]
Yes ... while preserving the positive impact on UTLS q ....

**Comment by adamrher** [comment]
@cecilehannay I wouldn't suggests stopping the simulation, but rather see it through to the end (10 years). This is our new control.

**Comment by adamrher** [comment]
ah OK I see the issue. So we want to try to crank up SWCF/LWCF a bit?
